server/controller/resources.js

```diff
@@ -0,0 +1,92 @@
+const Resource = require("../models/resource.js");
+module.exports = {
+  allResources: (req, res) => {
+    Resource.find()
+      .sort({ _id: -1 })
+      .then((resources) => res.status(200).json(resources));
```

Comment:
conrollers are usually classes,
can you try using  the class syntax instead of modules.

Reply:
thank you for review , i will research and try to change  class syntax.